### PR TITLE
N1- connect with mysql

### PR DESCRIPTION
**Pull request by dmkofgl**
Create spring boot project and configure connection to mysql database.

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/NewsPortal?useSSL=false&createDatabaseIfNotExist=true
```
Comment (avinahradau) [inline src/main/resources/application.properties]: why do you need this properties ? 

> ?useSSL=false&createDatabaseIfNotExist=true

Comment (dmkofgl) [reply src/main/resources/application.properties]: About createDatabaseIfNotExist=true 
If schema 'NewsPortal' doesn't exist thrown SQLSyntaxErrorException: Unknown database 'newsportal'
About useSSL=false
Have exception like this https://github.com/brettwooldridge/HikariCP/issues/1268.

Comment (avinahradau) [reply src/main/resources/application.properties]: > About createDatabaseIfNotExist=true
If schema 'NewsPortal' doesn't exist thrown SQLSyntaxErrorException: Unknown database 'newsportal'

Try to create database before starting this schema. I believe all this properties will not be needed

Comment (dmkofgl) [reply src/main/resources/application.properties]: I'm agree. There are no problems If use another jdk and database already exist.

**Pull request merged**